pkg/gui/controllers/staging_controller.go

```diff
@@ -227,7 +231,8 @@ func (self *StagingController) editHunk() error {
 
 	hunk := state.CurrentHunk()
```

Comment:
I think it would be more natural for users to invert the editHunk patch on the staging panel as before.

Reply:
Maybe, but we'd have to reintroduce the logic for building patches in reverse just for this very niche feature, and I don't think it's worth it. As I said on discord, I don't think anyone will ever use the editHunk feature in the staging panel anyway, so personally I don't think it should even be offered there.

Reply:
🙆

Reply:
I agree: let's try to keep the code as simple as possible given known use cases. If somebody raises an issue asking us to add the feature, we can consider it then. I intend to refactor all this code to make it much more readable so if/when that happens it should be an easier change.